users/stdiscosrv.rst

```diff
@@ -120,28 +111,80 @@ from clients there are three options:
   authenticate the server based on it's certificate and domain name.
 
 - Use any certificate pair and let clients authenticate the server based on
-  it's "device ID" (similar to Syncthing-to-Syncthing authentication). In
-  this case, using ``syncthing -generate`` is a good option to create a
-  certificate pair.
+  it's "device ID" (similar to Syncthing-to-Syncthing authentication). This
+  option can be used with the certificate automatically generated by the
```

Comment:
s/it's/its/
Maybe make Syncthing-to-Syncthing authentication a link to https://docs.syncthing.net/dev/device-ids.html#connection-establishment